src/tsig/interop.rs

```diff
@@ -36,7 +36,7 @@ fn tsig_client_nsd() {
     let rng = SystemRandom::new();
 
     let cur_dir = env::current_dir().unwrap();
```

Comment:
Just re-tested this, it would seem that this is set to the directory of `Cargo.toml` before executing the tests. However [rust book](https://doc.rust-lang.org/book/ch11-02-running-tests.html#running-tests-in-parallel-or-consecutively) doesn't consider the working directory to be relied on, so I'll add a PR to do this relative to CARGO_MANIFEST_DIR per [shepmasters SO answer](https://stackoverflow.com/a/30004252).